frontend/src/components/AppFooter.vue

```diff
@@ -23,6 +23,11 @@
               Voir le code source
               <v-icon small color="grey darken-3" class="ml-1">mdi-open-in-new</v-icon>
             </a>
+            . Vous pouvez également obtenir des informations concernant la
+            <a href="https://updown.io/2l7f" class="grey--text text--darken-3">
```

Comment:
Ah, aucun de nos liens avec l'icon open-in-new ici ouvrent vraiment dans un nouveau onglet - tu peux ajouter target blank et rel noopener ?

Reply:
Merci, bien vu. J'en ai ajouté pour tous les liens ayant l'icône open-in-new